src/Control/Eff/Gitter.hs

```diff
@@ -0,0 +1,37 @@
+{-
+    Notifaika reposts notifications
+    from different feeds to Gitter chats.
+    Copyright (C) 2015 Yuriy Syrovetskiy
```

Comment:
I thought, I may add myself as an author of the module, but not as a copyright holder, and here is no "author" field.